app/services/audit/logger.rb

```diff
@@ -0,0 +1,22 @@
+module Audit
+  class Logger
+    REDACTED_KEYS = %w[authenticity_token utf8 commit].freeze
+
+    def self.log(listener, user, data = nil)
+      Audit::Notification.notify(listener) do |payload|
+        payload.user_id = user.id
+        payload.roles = user.roles.pluck(:name)
+        payload.slug = caller_locations(1, 1)[0].label
```

Comment:
I suggest we add a comment the line before to explain what `caller_locations(1, 1)[0].label` means as it's not really intuitive...